### PR TITLE
Hw 1

### PR DESCRIPTION
**Pull request by A1exenderS**

### CODE REVIEW

File: tests/firstLoginTests.spec.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+const { test, expect } = require("@playwright/test");
+
+test.describe("two simple tests", () => {
+  test.beforeEach(async ({ page }) => {
+    const emailInput = page.getByPlaceholder("Username");
+    const passwordInput = page.getByPlaceholder("Password");
+    const loginButton = page.locator("#login-button");
+
+    await page.goto("https://www.saucedemo.com/");
```
Comment (4rtemOv) [inline tests/firstLoginTests.spec.js]: you can add this url to playwright config file

File: tests/firstLoginTests.spec.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+const { test, expect } = require("@playwright/test");
+
+test.describe("two simple tests", () => {
+  test.beforeEach(async ({ page }) => {
```
Comment (4rtemOv) [inline tests/firstLoginTests.spec.js:4]: well done using beforeEach 👍

File: tests/firstLoginTests.spec.js (hunk 1 above)
Comment (4rtemOv) [reply tests/firstLoginTests.spec.js]: Additionally as we have more time this week. Lets try to play and run tests on other browsers

**Review by 4rtemOv — CHANGES_REQUESTED** [review]

File: tests/firstLoginTests.spec.js (hunk 1 above)
Comment (A1exenderS) [reply tests/firstLoginTests.spec.js]: I tried :) just turned them off, for quicker test-running.

> Additionally as we have more time this week. Lets try to play and run tests on other browsers.

**Comment by AlinaZhykharieva** [comment]
Good job! I`ve run your tests
image

**Pull request merged**